samples/display-gaming-interstitial-ad/sample.ts

```diff
@@ -41,47 +46,62 @@ function defineGamingInterstitialSlot() {
   // Slot returns null if the page or device does not support interstitials.
   if (gamingInterstitialSlot) {
     gamingInterstitialSlot.addService(googletag.pubads());
+    if (!delay) {
+      googletag.display(gamingInterstitialSlot);
+    }
     printStatus('Waiting for interstitial to be ready...');
-
-    // Add event listener to register click handler once interstitial loads.
-    // If this event doesn't fire, check the browser console for errors.
-    googletag.pubads().addEventListener(
-        'gameManualInterstitialSlotReady',
-        (slotReadyEvent:
-             googletag.events.GameManualInterstitialSlotReadyEvent) => {
-          if (gamingInterstitialSlot === slotReadyEvent.slot) {
-            printStatus('Interstitial is ready.');
-
-            const button = document.getElementById('trigger')!;
-            button.style.display = 'block';
-            button.addEventListener(
-                'click',
-                () => {
-                  slotReadyEvent.makeGameManualInterstitialVisible();
-                  printStatus('Interstitial is active.');
-                },
-                {once: true},
-            );
-          }
-        });
-
-    googletag.pubads().addEventListener(
-        'gameManualInterstitialSlotClosed', resumeGame);
+  }
+  else {
```

Comment:
Code style: else should be inline with the if's closing brace.

Reply:
Fixed